packages/embark-test-runner/src/reporter.js

```diff
@@ -1,7 +1,30 @@
 class Reporter {
-  constructor() {
+  constructor(embark) {
+    this.embark = embark;
+
     this.passes = 0;
     this.fails = 0;
+    this.gasAccumulator = 0;
+
+    this.wireGasUsage();
+  }
+
+  wireGasUsage() {
+    const {events} = this.embark;
+    events.on('blockchain:proxy:response', (params) => {
```

Comment:
Nice. Instead of using web3, you use the proxy!
It's smart, because that way we are not dependent on the fact that the user is using WS or not 👍